core/include/usFrameworkEvent.h

```diff
@@ -133,47 +134,50 @@ class US_Core_EXPORT FrameworkEvent
   ~FrameworkEvent();
 
   /**
-   * Can be used to check if this FrameworkEvent instance is valid,
-   * or if it has been constructed using the default constructor.
+   * Returns <code>false</code> if the FrameworkEvent is empty (i.e invalid) and
+   * <code>true</code> if the FrameworkEvent is not null and contains valid data.
+   */
+  bool operator!() const;
```

Comment:
The other event classes have an explicit boolean conversion operator. Can we have the same here instead?